project-resources/github-continuity-policy.md

```diff
@@ -0,0 +1,31 @@
+# OpenJS Github Continuity Policy
+
+## Summary
+
+This policy is intended to ensure that OpenJS GithHub projects remain accessible and manageable. 
+
+It addresses the following use cases:
+
+- Archiving projects when maintainers are no longer reachable
+- Managing projects and users when maintainers are inactive or unreachable
+- Adding or removing admins in emergency situations
+
+## Continuity Options for Projects
+
+### Option 1: Add The Executive Director As An Admin
+
+If you select this option, you will add the OpenJS Foundation executive director (currently https://github.com/rginn) 
+as an admin to your project.
+
+### Option 2: Add Your Organization To The OpenJS Enterprise
+
+If you select this option, you will add your Organization to the OpenJS Foundation enterprise account. Doing so gives 
+the foundation the option to become an organization owner if necessary.  The admins of the enterprise account are 
+currently limited to the [Executive Director](https://github.com/rginn) 
+and [Director Of Program Management](https://github.com/bensternthal). 
+
+### Option 3: Opt-out
+
+Projects that feel their governance is sufficient to provide continuity may opt out of this policy by requesting an 
+exception from the CPC. This can be done by filing an issue in the 
+[CPC repository](https://github.com/openjs-foundation/cross-project-council/issues).
```

Comment:
Will there be an associated template for this opt-out request?

Reply:
Probably not, at least not intially :) I think we want some narrative on the project's decision. If it turns out there is a common set of questions folks ask in response to the opt-out request, we can capture that in a template.